Sources/Rendering/Core/Coordinate/Constants.d.ts

```diff
@@ -0,0 +1,14 @@
+export declare enum Coordinate {
```

Comment:
I would name it `CoordinateSystems` for consistency.

Reply:
That's what we have in Constants.js

Reply:
fair enough